lms/templates/navigation.html

```diff
@@ -180,6 +159,34 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
       </%block>
     </ol>
     % endif
+    % if static.show_language_selector():
+     <% languages = released_languages() %>
+        % if len(languages) > 1:
+        <ol class="user">
+            <li class="primary">
+              <form action="/i18n/setlang/" method="post" class="settings-language-form" id="language-settings-form">
+                <input type="hidden" id="csrf_token" name="csrfmiddlewaretoken" value="${csrf_token}">
+                % if is_user_authenticated:
+                <input title="preference api" type="hidden" id="preference-api-url" class="url-endpoint" value="${reverse('preferences_api', kwargs={'username': user.username})}" data-user-info="${is_user_authenticated}">
```

Comment:
It would be better to rename data-user-info to something that better-describes the information it contains -- such as data-user-authenticated, or data-user-auth-status, etc.